mathjax3-ts/adaptors/liteAdaptor.ts

```diff
@@ -112,6 +112,20 @@ export class LiteAdaptor extends AbstractDOMAdaptor<LiteElement, LiteText, LiteD
         return doc.root;
     }
 
+    /*
+     * @override
+     */
+    public getStatus(doc: LiteDocument) {
+        return '';  // statusline is not maintained
+    }
+
+    /*
+     * @override
+     */
+    public setStatus(doc: LiteDocument, status: string) {
+        // statusline is not maintained
+    }
+
```

Comment:
Do we really want those? See my earlier comment.

Reply:
OK, I'll reimplement as a fixed-position display div, like version 2 uses.